ci-operator/config/open-cluster-management/cluster-backup-operator/open-cluster-management-cluster-backup-operator-main.yaml

```diff
@@ -0,0 +1,81 @@
+base_images:
+  open-cluster-management_builder_go1.16-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+  repo-brancher:
+    name: repo-brancher
+    namespace: ci
+    tag: latest
```

Comment:
Fast forward process has changed. These lines aren't needed. See comment below for more info.